README.md

```diff
@@ -118,100 +114,35 @@
-    Server: Apache
-    ETag: "078de59b16c27119c670e63fa53e5b51"
-    Content-Length: 23081
-    …
-
-The client may send an "If-None-Match" HTTP Header containing the last retrieved etag. If the content has not changed the server returns a status code 304 (not modified) and no response body.
-
-## 1.3 Updating Resources via HTTP PUT
-
-Whenever a resource offers the HTTP PUT method to be updated as a whole.
-
-This means that there is no partial update mechanism for objects but every PUT request is sending the whole object representation. PUT schemas may exclude server generated values that cannot be edited, such as creation dates or authors.
-
-## 1.4 Cross Origin Resource Sharing (CORS)
-
-To work with browser based API clients using [Cross Origin Resource Sharing (Cors)](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing), servers will put the `Access-Control-Allow'` headers in the response headers and specifiy who can access the servers Json resources. The client can look for these values and proceed with accessing the resources.
-
-In a CORS scenario, web clients expect the following headers:
-* `Access-Control-Allow-Headers: Authorization, Content-Type, Accept` to allow the `Authorization`, `Content-Type` and `Accept` headers to be used via [XHR requests](https://en.wikipedia.org/wiki/XMLHttpRequest)
-* `Access-Control-Allow-Methods: GET, POST, PUT, OPTIONS` to allow the Http methods the API needs
-* `Access-Control-Allow-Origin: example.com` to allow XHR requests from the `example.com` domain to the BCF server
-
-For example, Asp.Net applications in IIS need the following entries in their `web.config` file. `*` means the server allows any values.
-
-    <httpProtocol>
-        <customHeaders>
-            <add name="Access-Control-Allow-Headers" value="Content-Type, Accept, X-Requested-With,  Authorization" />
-            <add name="Access-Control-Allow-Methods" value="GET, POST, PUT, DELETE, OPTIONS" />
-            <add name="Access-Control-Allow-Origin" value="*" />
-        </customHeaders>
-    </httpProtocol>
-
-## 1.5 Http Status Codes
-
-The BCF API relies on the regular Http Status Code definitions. Good sources are [Wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) or the [HTTP/1.1 Specification](https://tools.ietf.org/html/rfc7231).
+## 1.1 Open CDE Foundation API
 
-Generally, these response codes shall be used in the API:
-* `200 - OK` for `GET` requests that return data or `PUT` requests that update data
-* `201 - Created` for `POST` requests that create data
+BCF API is a member of the Open CDE API family. All Open CDE APIs are united by a shared common API called [Open CDE Foundation API](https://github.com/buildingSMART/shared-common-API). 
+The foundation API specifies a small number of services and a few conventions that are common to all Open CDE APIs. All BCF API implementations must implement the Foundation API and follow its conventions and guidelines. 
```

Comment:
Should we add a note the Foundation 1.0 was released together with BCF 2.2 to make it clearer for new people implementing these standards?

Reply:
Resolved to suffice with a mention in the release notes.